packages/edit-post/src/store/actions.js

```
@@ -338,17 +315,17 @@ export function* requestMetaBoxUpdates() {
 
 	try {
 		// Save the metaboxes
-		yield apiFetch( {
+		await apiFetch( {
 			url: window._wpMetaBoxUrl,
 			method: 'POST',
 			body: formData,
 			parse: false,
 		} );
-		yield controls.dispatch( editPostStore, 'metaBoxUpdatesSuccess' );
+		dispatch.metaBoxUpdatesSuccess();
 	} catch {
-		yield controls.dispatch( editPostStore, 'metaBoxUpdatesFailure' );
+		dispatch.metaBoxUpdatesFailure();
```

Interesting that both these actions do the same thing: update the `isSavingMetaBoxes` flag. They could be merged into one. We are ignoring the errors.

Also, do they need to be a part of the public API? There is no use case for that. Just like `REQUEST_META_BOX_UPDATES`, they could be internal/private.